resources/i18n/it_IT/fdmextruder.def.json.po

```diff
@@ -124,74 +157,42 @@ msgctxt "machine_nozzle_id label"
 msgid "Nozzle ID"
 msgstr "ID ugello"
 
-msgctxt "machine_nozzle_offset_x label"
-msgid "Nozzle X Offset"
-msgstr "Offset X ugello"
-
-msgctxt "machine_nozzle_offset_y label"
-msgid "Nozzle Y Offset"
-msgstr "Offset Y ugello"
-
-msgctxt "machine_extruder_start_code description"
-msgid "Start g-code to execute when switching to this extruder."
-msgstr "Inizio codice G da eseguire quando si passa a questo estrusore."
-
-msgctxt "extruder_prime_pos_x description"
-msgid "The X coordinate of the position where the nozzle primes at the start of printing."
-msgstr "La coordinata X della posizione in cui l’ugello si innesca all’avvio della stampa."
-
-msgctxt "extruder_prime_pos_y description"
-msgid "The Y coordinate of the position where the nozzle primes at the start of printing."
-msgstr "La coordinata Y della posizione in cui l’ugello si innesca all’avvio della stampa."
-
-msgctxt "extruder_prime_pos_z description"
-msgid "The Z coordinate of the position where the nozzle primes at the start of printing."
-msgstr "Indica la coordinata Z della posizione in cui l’ugello si innesca all’avvio della stampa."
-
-msgctxt "extruder_nr description"
-msgid "The extruder train used for printing. This is used in multi-extrusion."
-msgstr "Treno estrusore utilizzato per la stampa. Utilizzato nell’estrusione multipla."
+msgctxt "machine_extruder_start_pos_x label"
+msgid "Extruder Start Position X"
+msgstr "X posizione avvio estrusore"
```

Comment:
Also this it should be "Posizione X avvio estrusore" because the position where extruter start is along X axis